native-gradle-plugin/src/main/java/org/graalvm/buildtools/gradle/internal/DefaultGraalVmExtension.java

```diff
@@ -79,7 +78,8 @@ public DefaultGraalVmExtension(NamedDomainObjectContainer<NativeImageOptions> na
         agentOpts.getEnabled().convention(false);
         agentOpts.getModes().getConditional().getParallel().convention(true);
         agentOpts.getMetadataCopy().getMergeWithExisting().convention(false);
-        agentOpts.getFilterableEntries().convention(Arrays.asList("org.gradle.", "org.junit."));
+        // TODO: replace with agent access filter
+        // agentOpts.getFilterableEntries().convention(Arrays.asList("org.gradle.", "org.junit."));
```

Comment:
Is this supposed to stay commented out?

Reply:
We were considering how to properly filter `org.gradle` and `org.junit` through `access-filter` file. Do you have any suggestion what is the best way to apply filter file for this purpose?

Reply:
I'm not sure what you mean by that. We already have `getAccessFilterFiles` where you can add more access filter files. Is the problem that you don't want to use `getFilterableEntries`, or that it hardcodes junit? If it's about hardcoding, I'm not sure how we can work around.

In any case, it's suspicious that this code is commented out, and that no tests are failing.

Reply:
The idea is to use `access-filter.json` for filtering entries in the config files. Therefore we can replace custom [manual post processing](https://github.com/graalvm/native-build-tools/pull/614/files#diff-8d46943d72fca2ee4ffb4ecafae60ef189596e0a95f4b2879ac495a4abf7f758) and use existing Native Image feature that does the same.